app/views/organizations/tasks/_form.html.erb

```diff
@@ -1,9 +1,9 @@
-<%= form_for [@pet, @task], remote: true do |f| %>
```

Comment:
Somewhat of a unrelated thing I noticed is that maybe the `tasks` folder should be under `pets`. I think we got confused (or I did) when I didn't realize that we have a `pets/tabs` section instead.

Reply:
I've moved some templates in the latest commit but I'm not sure if it's an improvement? Things are still kind of split up in multiple places. What do you think?